index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1</title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+    <header class = "header1">
```

Comment:
header1 - this is not the correct way to name classes - class names should be meaningful

Rename `header2` and `box1`  as well